### PR TITLE
tart: add page

### PR DESCRIPTION
**Pull request by marc-48k**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**: `0.36.x`

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution and welcome to tldr.

File: pages/osx/tart.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# tart
+
+> Build, run and manage macOS and Linux virtual machines (VMs) on Apple Silicon.
+> More information: <https://github.com/cirruslabs/tart>.
+
+- Pull a remote VM image:
+
+`tart pull {{acme.io/org/name:tag}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/tart.md:8]:
```suggestion
- Pull a specific remote VM image:

`tart pull {{acme.io/org/name:tag}}`
```

Optional fix: `specific` may sound better.

File: pages/osx/tart.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Clone a VM from a local or remote image source:
+
+`tart clone {{source-vm}} {{vm-name}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/tart.md:12]:
```suggestion
- Clone a specific VM from a local or remote image source:

`tart clone {{source-vm}} {{vm-name}}`
```


The same issue as above.

File: pages/osx/tart.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Create a new Mac VM from an ipsw file:
+
+`tart create --from-ipsw={{[latest|path/to/file.ipsw]}} {{vm-name}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/tart.md:16]:
```suggestion
- Create a new Mac VM from a specific ipsw file:

`tart create --from-ipsw={{latest|path/to/file.ipsw}} {{vm-name}}`
```

Almost the same issue as above.
Mandatory fix: `[` and `]` must be removed as not specified syntax in style guide.

File: pages/osx/tart.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Run an existing VM:
+
+`tart run {{vm-name}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/tart.md:20]:
```suggestion
- Run a specific existing VM:

`tart run {{vm-name}}`
```

Almost the same issue as above.

File: pages/osx/tart.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Run an existing VM with a mounted directory:
+
+`tart run --dir={{directory}}:{{/path/to/local/directory}} {{vm-name}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/tart.md:24]:
```suggestion
- Run an existing VM with a specific mounted directory:

`tart run --dir={{path/to/directory}}:{{/path/to/local_directory}} {{vm-name}}`
```

Mandatory fix: [path format is incorrect](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).

File: pages/osx/tart.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- List VMs:
+
+`tart list`
+
+- Get IP address of a running VM:
+
+`tart ip {{vm-name}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/tart.md:32]:
```suggestion
- Get IP address of a specific running VM:

`tart ip {{vm-name}}`
```

Optional fix: `specific` may sound better.

File: pages/osx/tart.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- Change a VM's display resolution:
+
+`tart set {{vm-name}} --display {{width}}x{{height}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/tart.md:36]:
```suggestion
- Change a specific VM's display resolution:

`tart set {{vm-name}} --display {{width}}x{{height}}`
```
The same issue as above.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

File: pages/osx/tart.md (hunk 4 above)
Comment (kbdharun) [reply pages/osx/tart.md:20]: `a specific` won't apply here. The original sentence (Run an existing VM) is fine.

File: pages/osx/tart.md (hunk 1 above)
Comment (marc-48k) [reply pages/osx/tart.md:8]: My pleasure! thanks for reviewing.

I think I'd prefer not to make the optional "specific" changes. I've deliberated on it for a while and, on reflection, I think it reads better as it is.

Comment (EmilySeville7cfg) [reply pages/osx/tart.md:8]: That's why they are optional - you can do with them what you want. 😄

**Review by EmilySeville7cfg — APPROVED** [review]

**Pull request merged**